packages/auth/package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "auth",
+  "scripts": {
+    "lint": "yarn lint:local",
+    "format": "yarn format:local",
+    "format:check": "yarn format:check:local",
+    "knex": "knex",
+    "build": "yarn clean && tsc --build tsconfig.json",
+    "clean": "rm -fr dist/",
+    "test": "echo \"Error: no test specified\" && exit 1",
```

```suggestion
    "test": "yarn test:local",
```
and add a [`jest.config.js`](https://github.com/interledger/rafiki/blob/main/packages/rates/jest.config.js)